openvdb/openvdb/points/PrincipleComponentAnalysis.h

```diff
@@ -0,0 +1,148 @@
+// Copyright Contributors to the OpenVDB Project
```

Comment:
It's actually PrincipalComponentAnalysis, instead of Principle.

Reply:
Oof thanks @apradhana. I'm not a big fan of this header name as there's obviously lots of different types of PCA, this is just one application of it in a very specific way, but I renamed it from something I didn't like either... so I'll fix this, but want to have a better think about the naming here